back/src/Guide/Cli.hs

```diff
@@ -0,0 +1,75 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell   #-}
+
+-- | This module adds command line interface to run server.
+--
+-- See help with 'guide --help' to see available commands.
+module Guide.Cli
+       ( Command (..)
+       , parseCli
+       ) where
+
+
+import Imports
+
+import Data.Version (showVersion)
+import Development.GitRev (gitCommitDate, gitDirty, gitHash)
+import Options.Applicative (Parser, ParserInfo, command, execParser, fullDesc, help, helper, info,
+                            infoHeader, infoOption, long, metavar, progDesc, short, strOption,
+                            subparser, (<|>))
+import Options.Applicative.Help.Chunk (stringChunk)
+
+import Paths_guide (version)
+
+import qualified Data.Text as T
+
+
+-- | All available commands
+data Command
+    = Guide               -- ^ run server
+    | DryRun              -- ^ load database and exit
+    | LoadPublic String   -- ^ load PublicDB, create base on it and exit
+
+----------------------------------------------------------------------------
+-- Parsers
+----------------------------------------------------------------------------
+
+-- | Main parser of the application.
+parseCli :: IO Command
```

Comment:
Can you post the output of `guide --help`? And, say, `guide load-public --help`?

Reply:
```
$ guide --help

Guide is wiki

Usage: guide [-v|--version] [COMMAND]


Available options:
  -h,--help                Show this help text
  -v,--version             Show Guide version

Available commands:
  run                      Run server
  dry-run                  Load database and exit
  load-public              Load PublicDB, create base on it and exit

NOTE:
Command 'guide' is the same as 'guide run'

----------------------------------------------------------------------------

$ guide load-public --help
Usage: guide load-public (-p|--path FILEPATH)
  Load PublicDB, create base on it and exit

Available options:
  -h,--help                Show this help text
  -p,--path FILEPATH       Public DB file name
```